includes/html/pages/device.inc.php

```diff
@@ -267,6 +267,11 @@
             $routing_tabs[] = 'cisco-otv';
         }
 
+        $device_routing_count['routes'] = @dbFetchCell('SELECT COUNT(*) FROM `inetCidrRoute` WHERE `device_id` = ?', array($device['device_id']));
```

Comment:
Why are you suppressing errors here?

Reply:
I copied paste the code from VRFs a few lines above ... did not even notice the @ while doing it I suppose.